### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by devAmlan**
This is the README of my project,Please review my PR.

### CODE REVIEW

**Review by developer-anand — APPROVED** [review]
Good to Merge

**Review by archit7448 — APPROVED** [review]
good to merge

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Now build awesome UIs  with Mlan ui.
+
+### Make your web pages beautiful and responsive using Mlan UI, without much hassle in css
+---
+**How to use MLAN UI?** 
+- To use MLAN UI in your project you need to copy and paste the below link, and add this inside the <head> tag of your HTML file OR you can import the code into your CSS file.
+
+> `<link rel="stylesheet" href="https://mlanui.netlify.app/library/style.css" />`
+
+> `@import url("https://mlanui.netlify.app/library/style.css")`
```
Comment (sudipkundu999) [inline README.md:10]: try wrapping the code in triple backticks then you'll get a better looking code block with a copy button.
example - 
codereview

output
```html
<link rel="stylesheet" href="https://mlanui.netlify.app/library/style.css" />
```
```css
@import url("https://mlanui.netlify.app/library/style.css")
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Input
+**We have 3 types on Input in this library.You can use it while making forms in your projects.**
+
+- Text box 
+- general input box 
+- Input box with error message
+
+## List
+**There are 4 types of List (general unordered list, circle list and no stye list) also a Stacked list (can be used to show notification on your project)**
```
Comment (sudipkundu999) [inline README.md:46]: Follow a common pattern, if you're showing types using lists follow that only

File: README.md (hunk 1 above)
Comment (devAmlan) [reply README.md:10]: I will warp the code , thanks sudip

**Pull request merged**